tee_deployment/configs/kms.env

```diff
@@ -7,15 +7,27 @@ APP_NAME="launcher_test_app"
 #VMM_RPC=unix:../../../build/vmm.sock
 VMM_RPC=http://127.0.0.1:16000
 
-# The address of the dstack guest agent service listening on Host machine
-GUEST_AGENT_ADDR=127.0.0.1:9206
-
-# Port use on the host machine to SSH into the CVM  (only for dev images)
-SSH_HOST_PORT=127.0.0.1:9207
-# The port on the host machine used by the MPC app to listen for incoming connections
-MPC_PUBLIC_PORT=0.0.0.0:4444
-# The port in the CVM used by the MPC app to listen for incoming connections
-MPC_VM_PORT=4444
+
+# Address of the dstack guest agent service inside the CVM
+INTERNAL_AGENT_ADDR=127.0.0.1:9206
```

Comment:
Just a question. Why do we need internal port configurations? Is it because we expect they might change at some point. Afaik they cannot change for a given dstack image, right?